recipes/xraylib/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "3.1.0" %}
+
+package:
+  name: xraylib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tschoonj/xraylib
+  git_rev: xraylib-{{ version }}
```

Comment:
Ok, happy to save time on CI servers.  This seems like an inconsistently applied policy, should it become part of the linter?